smart_pet_buddy/lib/edit_profile.dart

```diff
@@ -0,0 +1,217 @@
+                    ),
+                    Positioned(
+                        bottom: 0,
+                        right: 0,
+                        child: GestureDetector(
+                          onTap: () async {
+                            PickedFile image = await ImagePicker().getImage(source: ImageSource.gallery);
+                            print(image.toString());
+                            TaskSnapshot snapshot = await FirebaseStorage.instance.ref().child('user_image/${user.uid}').putFile(File(image.path));
+                            String url = await snapshot.ref.getDownloadURL();
+
+                            user.updateProfile(photoURL: url).whenComplete(() async {
+                              print('Successful photo change');
+                              user = FirebaseAuth.instance.currentUser;
+                              setState(() {});
+                            });
+                          },
+                          child: Container(
+                            height: 40,
+                            width: 40,
+                            decoration: BoxDecoration(
+                              shape: BoxShape.circle,
+                              border: Border.all(
+                                width: 4,
+                                color: Theme.of(context).scaffoldBackgroundColor,
+                              ),
+                              color: Colors.green,
+                            ),
+                            child: Icon(
+                              Icons.edit,
+                              color: Colors.white,
+                            ),
+                          ),
+                        )),
+                  ],
+                ),
+              ),
+              SizedBox(
+                height: 35,
+              ),
+              buildTextField("Full Name", "Dor Alex", nameController),
+              buildTextField("E-mail", "alexd@gmail.com", emailController),
+              PasswordInput(labelText: "Password", placeholder: "********", textController: passwordController,),
+              buildTextField("Location", "", locationController),
+              SizedBox(
+                height: 35,
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  OutlineButton(
+                    padding: EdgeInsets.symmetric(horizontal: 50),
+                    shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.circular(20)),
+                    onPressed: () {
+                      Navigator.of(context).pop();
+                    },
+                    child: Text("CANCEL",
+                        style: TextStyle(
+                            fontSize: 14,
+                            letterSpacing: 2.2,
+                            color: Colors.black)),
+                  ),
+                  RaisedButton(
```

Comment:
We could use the "ElevatedButton" here since RaisedButton seems to be deprecated?